pkg/transformer/kubernetes/kubernetes.go

```diff
@@ -640,9 +640,10 @@ func (k *Kubernetes) Undeploy(komposeObject kobject.KomposeObject, opt kobject.C
 			//FIXME: gracePeriod is nil
 			err = rpDeployment.Stop(namespace, t.Name, TIMEOUT*time.Second, nil)
 			if err != nil {
-				return err
```

Comment:
Just create a list of errors and then return that maybe and the caller of undeploy should print it in right way or whatever!